group_vars/figgy_staging.yml

```diff
@@ -156,8 +156,6 @@ rails_app_vars:
     value: 'https://iiif-cloud-staging.princeton.edu/iiif/2/'
   - name: OCR_IN_PATH
     value: '/mnt/illiad/ocr_scan'
-  - name: OCR_OUT_PATH
```

Comment:
Why is this no longer configured?

Reply:
Thanks for catching that. I went back and forth on this while in development. Env vars are now set for both prod and staging.